lib/scraped_page_archive.rb

```diff
@@ -14,41 +14,100 @@
 class ScrapedPageArchive
   class Error < StandardError; end
 
-  attr_writer :github_repo_url
+  class GitStorage
```

Comment:
Yep good point. Done in https://github.com/everypolitician/scraped_page_archive/commit/083ac2a9baa100a0d6879ac2e4973b587668907f.